core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -118,7 +125,7 @@ private case class Context(
 
     def toActionArgument(user: Option[Identity], boxQueryAndBody: Boolean): Map[String, JsValue] = {
         val queryParams = if (boxQueryAndBody) {
-            Map(propertyMap.query -> JsString(query.render(new StringRendering, StandardCharsets.UTF_8).get))
+            Map(propertyMap.query -> JsString(query.toString))
```

Comment:
Looked at the code, the render method wasn't removed, it was made private. Here's what the toString method does: ```
    override def toString = UriRendering.QueryRenderer.render(new StringRendering, this).get
```